grafana-dashboard.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  graph_id_base = 100
+  apiary_data_buckets = 2
+}
+
+data "template_file" "graphs" {
+  template = file("${path.module}/templates/graph.tpl")
+  count = length(local.apiary_data_buckets)
+  vars = {
+    bucket_name = local.apiary_data_buckets[count.index]
+    title_bucket_name = local.apiary_managed_schema_names_replaced[count.index]
+    graph_id = range(local.graph_id_base, length(local.apiary_data_buckets) * local.apiary_data_buckets, local.apiary_bucket_prefix)[count.index]
+  }
+}
+
+data "template_file" "dashboard" {
+  template = file("${path.module}/templates/dashboards.tpl")
+  vars = {
+    panels = "[${join(",", data.template_file.graphs.*.rendered)}]"
+  }
+}
+
+resource "kubernetes_config_map" "grafana-dashboard" {
```

Comment:
typicially we use `_` in terraform resource names and `-` for AWS resources like S3 bucket names...etc.
so can you use `grafana_dashboard` instead?